azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -81,22 +83,39 @@ ServiceClientCredentials getCredentials() {
             final String msiEndpoint = getProperty(this.environment, "MSI_ENDPOINT");
             final String msiSecret = getProperty(this.environment, "MSI_SECRET");
             return new AppServiceMSICredentials(AzureEnvironment.AZURE, msiEndpoint, msiSecret);
-        } else if (this.environment.containsProperty(Constants.AZURE_CLIENTID)
+        }
+
+        final long timeAcquiringTimeoutInSeconds = this.environment.getProperty(
```

Comment:
Is this property required?

Reply:
It's a acquire token timeout setting, default value is provided, so no such property is not a problem here.

Reply:
My point here is if user doesn't fill such property, will this return 0 or exception?

Reply:
It will return `Constants.TOKEN_ACQUIRE_TIMEOUT_SECS`, as the called method here is: `<T> T getProperty(String key, Class<T> targetType, T defaultValue);`